packages/accordion/src/system-overrides.css

```diff
@@ -0,0 +1,320 @@
+/*
```

Comment:
Shouldn't this be imported by by `accordion.css`?

Reply:
yess!!! thnx for the catch